templates/index.html

```diff
@@ -5,7 +5,7 @@ <h1>Login Proof of concept</h1>
 </form>
 
 <div>
-  <a href="/logged">Go to super protected page</a>
+  <a href="/app-detail">Go to super protected page</a>
```

Comment:
You could probably use `<a href="{{ url_for('get_app_detail') }}">`. Avoids mismatching URLs later.

(I realise this is mostly a test file and probably doesn't matter much on it)